vmtop.py

```diff
@@ -1373,6 +1407,31 @@ def main():
             signal_map={
             signal.SIGTERM: exit_gracefully
         }):
+            # Daemonizing kills all the previously opened sockets, so
+            # run prometheus after daemonizing
+            if args.prometheus:
+                try:
+                    from prometheus_client import start_http_server
+                    import socket
+                    host, port = args.prometheus.split(':')
+                    start_http_server(int(port), host)
```

Comment:
do we really need to do that multiple times ?
I think once in the main is enough.

Reply:
I think all of the "try to import and possibly throw exception" code flow is a anti-pattern. All the imports should be at the top of the file. If something can't be imported set a global variable/flag and check that.

Reply:
yes I agree with that

Reply:
I will merge this PR because it's unrelated to that issue, and we'll make another PR to clean that up